File: test61_out/test61/Led/Led.js

/*
 * ======== MODULE test61.Led ========
 */

var $$c = function() {
    const Led = {};
    Led.Led = Led
    Led.$name = 'test61.Led';
    Led.pollen$used = 0;
    Led.targetInit = new $$Ref('test61_Led_targetInit__I');
    Led.on = new $$Ref('test61_Led_on__E');
    Led.off = new $$Ref('test61_Led_off__E');
    Led.toggle = new $$Ref('test61_Led_toggle__E');
    Led.$$hostInit = function() {
        var $$text = '';
        return $$text;
    }
    Led.pollen__uses__ = function() {
    }
    Led.$$privateInit = function() {
        Led.pin = 0;
    }
    return Led;
}
var $$u = $$c();
$units['test61.Led'] = $$u;
$units.push($$u);

File: test61_out/test61/Led/Led.c

/*
 * ======== string literals ========
 */

static const char test61_Led__s__fileName[] = "Led.p";
static const char test61_Led__s__0[] = "Led";

/*
 * ======== function definitions ========
 */

void test61_Led_toggle__E() {
    test61_AvrPin_toggle__E();
}

void test61_Led_targetInit__I() {
    test61_AvrPin_clear__E();
    test61_AvrPin_makeOutput__E();
}

void test61_Led_off__E() {
    test61_AvrPin_clear__E();
}

void test61_Led_on__E() {
    test61_AvrPin_set__E();
}

